lib/index.js

```diff
@@ -1,33 +1,46 @@
 var foodsRequests = require('./requests/foods')
 var foodsDiary = require('./requests/diaryFoods')
 var events = require('./eventListeners/eventListeners')
+var fileName = location.pathname.split('/').slice(-1)[0]
 require('./foods')
 require('./diary')
 
 $(document).ready(() => {
+  renderData(fileName)
```

Comment:
Clean refactor for doc.ready! Later iterations we'll pull further into separate files.